woff2/index.html

```diff
@@ -1845,12 +1845,14 @@ <h3 id="normative-references"><span class='secno'>D.1 </span>Normative Reference
           version of the WOFF specification is available at <a href="https://www.w3.org/TR/WOFF/">http://www.w3.org/TR/WOFF/</a><br>
         </cite></dd>
       <dt id="ref-OFF">[OFF]</dt>
-      <dd><cite><a href="http://standards.iso.org/ittf/PubliclyAvailableStandards/c066391_ISO_IEC_14496-22_2015.zip">Open
-            Font Format specification</a> (ISO/IEC 14496-22:2015).</cite> together with
-          <cite><a href="http://standards.iso.org/ittf/PubliclyAvailableStandards/c069450_ISO_IEC_14496-22_2015_Amd_1_2017.zip">
-            Amendment 1: Updates for font collections functionality</a></cite> and
+      <!-- status at https://www.iso.org/standard/74461.html -->
+      <dd><cite><a href="https://standards.iso.org/ittf/PubliclyAvailableStandards/c074461_ISO_IEC_14496-22_2019.zip">Open
+            Font Format specification</a> (ISO/IEC 14496-22:2019).</cite> together with
+          <cite><a href="https://standards.iso.org/ittf/PubliclyAvailableStandards/c078865_ISO_IEC_14496-22_2019_Amd_1_2020(en).zip">
+            Amendment 1: Color font technology and other updates</a></cite>
+            <!--  and
           <cite><a href="http://standards.iso.org/ittf/PubliclyAvailableStandards/c070994_ISO_IEC_14496-22_2015_Amd_2_2017.zip">
-            Amendment 2: Updated text layout features and implementations</a></cite></dd>
+            Amendment 2: Updated text layout features and implementations</a></cite></dd> -->
```

Comment:
Since we have updated the original reference to new, 4th edition (ISO/IEC 14496-22:2019), we should remove the reference to old amendment 2 (ISO/IEC 14496-22:2015_Amd2:2017) for 3rd edition - it's already been rolled into the 4th.

Reply:
Amd2 is commented out.
I left it there in case DAmd2 got ratified meanwhile.